### PR TITLE
feat(cost-analysis-page): connect route params with granularity options

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Connect route params with granularity options
And fix some bugs
- route query name of Cost analysis page
- type bug

### CODE REVIEW

**Review by WANZARGEN — APPROVED** [review]
LGTM

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue [hunk 1]
```diff
@@ -96,8 +97,9 @@ const setChartData = debounce(async () => {
     state.loading = true;
 
     const rawData = await listCostAnalysisData();
-    state.legends = getLegends<CostAnalyzeRawData>(rawData, costAnalysisPageState.granularity, costAnalysisPageState.chartGroupBy);
-    state.chartData = getXYChartData<CostAnalyzeRawData>(rawData, costAnalysisPageState.granularity, costAnalysisPageState.period, costAnalysisPageState.chartGroupBy);
+    const { period, granularity, chartGroupBy } = costAnalysisPageState;
+    state.legends = getLegends<CostAnalyzeRawData>(rawData, granularity, chartGroupBy);
+    state.chartData = period ? getXYChartData<CostAnalyzeRawData>(rawData, granularity, period, chartGroupBy) : [];
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue:102]: suggestion: How about checking `period` in watch()?

**Review by yuda110 — APPROVED** [review]
LGTM!! Please check my comment

File: apps/web/src/services/cost-explorer/cost-analysis/CostAnalysisPage.vue [hunk 2]
```diff
@@ -30,7 +30,7 @@ import type {
 
 
 interface Props {
-    querySetId?: string;
+    costQuerySetId?: string;
```
Comment (piggggggggy) [inline apps/web/src/services/cost-explorer/cost-analysis/CostAnalysisPage.vue:33]: praise: Thanks~!

**Review by piggggggggy — APPROVED** [review]
LGTM~!

**Pull request merged**